kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -140,6 +146,7 @@ def _get_labels_selector(self) -> str | None:
             return None
         return ",".join([f"{key}={value}" for key, value in enumerate(self.labels)])
 
+    @pytest.mark.usefixtures("mock_get_connection")
```

Comment:
Usefixtures makes little sense when put as method decorator (more typing, less explicit, only saves "unused parameter" in IDE. But it makes whole lot of sense when put on a class - can we please remove it it from the methods and put it once on TestKubernetesPodOperatorSystem instead?